applications/nrf5340_audio/broadcast_source/main.c

```diff
@@ -44,6 +44,60 @@ K_THREAD_STACK_DEFINE(le_audio_msg_sub_thread_stack, CONFIG_LE_AUDIO_MSG_SUB_STA
 
 static enum stream_state strm_state = STATE_PAUSED;
 
+/* Buffer for the UUIDs. */
+NET_BUF_SIMPLE_DEFINE_STATIC(uuid_data, CONFIG_EXT_ADV_UUID_BUF_MAX);
+
+/* Buffer for periodic advertising BASE data. */
+NET_BUF_SIMPLE_DEFINE_STATIC(base_data, 128);
+
+/* Extended advertising buffer. */
+static struct bt_data ext_adv_buf[CONFIG_EXT_ADV_BUF_MAX];
+
+/* Periodic advertising buffer. */
+static struct bt_data per_adv_buf;
+
+#if (CONFIG_AURACAST)
+/* Size of the Public Broadcast Announcement meta data bytes. This is given by the number of meta
+ * data items, multiplied by the sum of three octets (struct bt_data).
+ *
+ *  e.g. size = number_meta_data_items * sizeof(struct bt_data)
+ *
+ * @note: Make sure this large includes all meta data
+ *        (any additional meta data requires an increase of this value).
+ */
+#define BROADCAST_SRC_PBA_META_NUM	    (2)
+#define BROADCAST_SRC_PBA_META_SIZE	    (BROADCAST_SRC_PBA_META_NUM * sizeof(struct bt_data))
```

Comment:
This meta size should be calculated dynamically. The pba_data array should take some sort of config that is the max data we can put in, and then it's up to the user to decide how much of that is filled